src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/domain/FaktaOgVurderingMapper.kt

```diff
@@ -129,12 +129,18 @@ fun mapAktiviteterLæremidler(
 ): AktivitetLæremidler {
     return when (type) {
         AktivitetType.TILTAK -> TiltakLæremidler(
-            fakta = FaktaAktivitetLæremidler(prosent = faktaOgSvar.prosent!!),
+            fakta = FaktaAktivitetLæremidler(
+                prosent = faktaOgSvar.prosent!!,
+                studienivå = faktaOgSvar.studienivå, // TODO: Det er ikke obligatorisk å sende inn studienivå for tiltak av variant "Høyere utdanning". Må deale med det.
```

Comment:
Er det valgfritt? Burde det ikke være at den ikke sendes inn for "Høyere utdanning" ?

Reply:
Den skal preutfylles dersom det er variant høyere utdanning

Reply:
Fjernet TODO-en